includes/admin/class-wc-admin-notices.php

```diff
@@ -87,6 +88,7 @@ public static function reset_admin_notices() {
 		self::add_wc_admin_feature_plugin_notice();
 		self::add_notice( 'template_files' );
 		self::add_min_version_notice();
+		self::add_maxmind_missing_license_key_notice();
```

Comment:
what do you think about hooking this to `update_option_woocommerce_default_customer_address` in `init` method of this class as well. That way all our notice code will be in one place and we won't need to repeat this in later in `display_missing_license_key_notice` ?

Reply:
We run this method on `woocommerce_installed`, so it get triggered after the installation or any update. We plan to move this integration to a feature plugin in the future, so we plan to remove from this class, that's why we are not including all in one place right now.